src/app/users/update-business-map-info.ts

```diff
@@ -0,0 +1,20 @@
+import { AccountsRepository } from "@services/mongoose/accounts"
+
+export const updateBusinessMapInfo = async ({
+  username,
+  coordinate,
+  title,
+}: {
+  username: string
+  coordinate: { latitude: number; longitude: number }
+  title: string
+}): Promise<Account | Error> => {
+  const accountsRepo = AccountsRepository()
+
+  const account = await accountsRepo.findByUsername(username as Username)
+  if (account instanceof Error) return account
+
+  account.coordinate = coordinate as Coordinates
+  account.title = title as BusinessMapTitle
```

Comment:
if we will have business info associated to username then it shoud be part of user repo.